mailpoet/tests/_support/AcceptanceTester.php

```diff
@@ -876,6 +876,19 @@ public function triggerAutomationActionScheduler(): void {
     $i->cli(['action-scheduler', 'run', '--force']);
   }
 
+  public function triggerNewsletterScheduledTask($newsletterId) {
```

Comment:
I would suggest renaming the method to `rescheduleNewsletterToBeSentImmediatelly`. I think it better describes what it does and could be easier to understand in the test context. Does that sound reasonable?

Reply:
Noted